governance-charter.md

```diff
@@ -78,15 +78,15 @@ Exception Process outline below.
 - Representatives from major organizations that have adopted OpenTelemetry as
   end-users (one representative per organization)
 
-The bootstrap committee explicitly believes that this heuristic will be
+The Governance Committee explicitly believes that this heuristic will be
 inaccurate and not represent the entire community. An exception form will be
 provided for people who have contributed to the project but may not meet these
-criteria. Exception eligibility applications will be approved by the bootstrap
+criteria. Exception eligibility applications will be approved by the Governance
 committee by a simple majority vote. The exception process will be used as data
 points for refining the criteria for the future.
 
-Within its first year, the Governance Committee will redefine these criteria
-around a more robust idea of community membership.
+The Governance Committee commits to redefine these criteria around a more robust 
```

Comment:
IMO, what we have is robust enough, and providing the exception form has proven to be quite sufficient. I'd be more comfortable if we stated that we'd be re-evaluating the current criteria by the end of 2024.

Reply:
This would work for me as well, btw, k8s uses this wording

> It is the responsibility of the steering committee to refine these criteria prior to each election, including setting the number of required contributions, and adding any additional committee memberships that include eligibility.

I was thinking about an adjusted version of that

Reply:
Sounds good to me.

Reply:
updated, see the new version

I kept the piece with the "setting the number of required contributions", because at some point we _may_ consider raising that value above 20. I replaced the word "committee memberships" with "community membership" since this is referring to another sentence in the k8s document around code of conduct and other committees we do not have as of today.